guides/puppet-web-apps/authorization/README.md

```diff
@@ -0,0 +1,227 @@
+#
+Authorization Library
+
+This library helps the developer to prevent the user from accessing / acting on data he doesn't have privileges to.
+
+## Foundation
+
+### Permissions
+
+Permission is a core concept in Authorization. A Permission represents specific action that we can allow the user to perform. This could be "listing all users" or "displaying details of user Joozek" or "changing the details of user Joozek". The granularity of Permissions (you could have just one "changing the details of user X" or specific Permission for each property of the user) is up to the developer and it's recommended to limit the scope of permissions to an object (so "change user X" is probably better then "change any user" and "change last name of user X").
+
+Whenever a user will perform an action (open the page, click a button) a Permission will be checked to verify if he is allowed to do it. Grouping Permissions together ("show list of users or companies") is discouraged, because it's easy to group them later in the configuration (AuthorizationRules).
+
```

Comment:
Please reduce the number of parentheses, they make it really hard to understand the text